packages/preview/german-invoice/0.1.0/typst.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "german-invoice"
```

If the name isn't distinct enough
```suggestion
name = "german-fx-invoice"
```